code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -614,7 +616,14 @@ var/global/list/robot_modules = list(
 					LANGUAGE_SIIK_MAAS = 1,
 					LANGUAGE_SKRELLIAN = 1,
 					LANGUAGE_GUTTER = 1,
-					LANGUAGE_ROOTSONG = 1
+					LANGUAGE_VAURCESE = 0,
+					LANGUAGE_ROOTSONG = 1,

```

Comment:
Cause 1 makes them speak the language
```suggestion
					LANGUAGE_ROOTSONG = 0,
```

Reply:
They were able to speak it before, no reason to change it?

Reply:
Why would cyborg or AI speak rootsong?